src/components/container/App/App.js

```diff
@@ -1,10 +1,7 @@
-import './App.scss';
```

Comment:
Please revert all changes in App.js -> lets pretend any possible merge conflicts!